todo-list-connected-to-api/maryam/src/AddNewItem.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import {
+  makeStyles,
+  ListItem,
+  IconButton,
+  ListItemSecondaryAction,
+  TextField
+} from '@material-ui/core';
+import {Add} from '@material-ui/icons'
+// ----------------------------------------------------------------
+const useStyles = makeStyles((theme) => ({
+  form: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+// ----------------------------------------------------------------
+const AddNewItem = () => {
+    const classes = useStyles();
+    const [textFieldValue, setTextFieldValue] = useState('');
+
+    const addClickedHandler = async () => {
+        fetch(`https://todolist.ehsan-rafee.ir/api/todolist`,{
```

Comment:
__[HIGH]__  
Same problem as `TodoItem.js` exists here. You should pass `setTodoItems` to this component and update your state so React updates UI.